src/main/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{yyyy-MM-dd HH:mm:ss} %-5level %logger{36} - %msg%n
```

Comment:
Lets also add thread name and line number here
I also dont remember whether this will use the system timezone. If yes than lets explicitly specify utc

Reply:
Thread name, method name, line number, utc timezone were added in commit 1ac7ffc